swift/Tests/CoinAddressDerivationTests.swift

```diff
@@ -190,6 +190,8 @@ class CoinAddressDerivationTests: XCTestCase {
                 case .filecoin:
                     let expectedResult = "f1zzykebxldfcakj5wdb5n3n7priul522fnmjzori"
                     AssetCoinDerivation(coin, expectedResult, derivedAddress, address)
+                @unknown default:
+                    fatalError()
```

Comment:
can you remove this?

Reply:
we need to make sure each coin is covered

Reply:
This is a test, to suppress one swift compiler warning, if you don't implement new case, the test will crash anyway

Reply:
Any other way to fix this? It's better UX when compiler tell you to fill this field rather than crash when running tests